test-app/src/test/resources/application-test.yml

```diff
@@ -60,3 +60,7 @@ hmpps.sqs:
       arn: arn:aws:sns:eu-west-2:000000000000:${random.uuid}
     fifotopic:
       arn: arn:aws:sns:eu-west-2:000000000000:fifotopictestapp.fifo
+  buckets:
+    bucket:
+      access_key_id: foobar
+      secret_access_key: foobar
```

Comment:
You might need a `${random.uuid}` here. This solves an issue where Spring Boot re-uses the same Localstack resources when running integration tests locally (e.g. queues, buckets). Things like queue based tests interfering with each other when running tests in parallel, for example.